app/javascript/mastodon/features/ui/index.js

```diff
@@ -128,11 +133,18 @@ class UI extends React.PureComponent {
         </ColumnsArea>
       );
     } else {
+      // TODO: Better check & error handling
+      if (Object.keys(staticColumns).length !== 3) {
```

Comment:
Because there are three columns. Refactored to be cleared and not rely on a magic constant.